### ISSUE TITLE
Bad performance on real-shot images in BDD100k

### ISSUE THREAD

**Issue by lgz-0713** [ykdai/Flare7K#9]
I downloaded the provided pretrained Uformer model from the authors and performed inference on the real-shot nighttime images in the BDD100k dataset. However, it did not work. I am unsure whether I did not set the right hyper-parameters or if it was limited by the domain gap between real flares and synthesized flares.

![real_night_0](https://github.com/ykdai/Flare7K/assets/53067136/b7cbdd96-2ae6-47d9-954c-6ec7966102d3)
![deflare_img_0](https://github.com/ykdai/Flare7K/assets/53067136/1c3d2f87-76bc-40ad-842f-6d40eb11bb61)

**Comment by lgz-0713** [comment]
OK, it works when I give up normalizing the input.
